adding targetURL when using `status` action (#118)
gagoar/use-herald-action: README.md

| `customMessage`   |        `string`        |    No    | Message to be commented on the pull request when the rule is applied (requires `action === comment`)                                                                                                                                                     |
| `users`           |       `string[]`       |    No    | GitHub user handles (or emails) on which the rule will take action. It will not be used when `action` is set to `comment` and `customMessage` field is present                                                                                           |
| `teams`           |       `string[]`       |    No    | GitHub teams on which the rule will take action. It will not be used when `action` is set to `comment` and `customMessage` field is present                                                                                                              |

> | `targetURL`       |        `string`        |    No    | It will be used when `action` is set to `status` as the link under [Details](https://github.blog/2012-09-04-commit-status-api/)                                                                                                                          |

Reviewer: ```suggestion
| `targetURL`       |        `string`        |    No    | When `action` set to `status`, link to which the [Details](https://github.blog/2012-09-04-commit-status-api/) link will point                                                                                                                          |
```
